packages/asset-uploader/package.json

```diff
@@ -37,5 +37,13 @@
     "!*.test.*"
   ],
   "license": "MIT",
-  "private": false
+  "private": false,
+  "tsup": {
```

Comment:
I had to move the tsup-related config somewhere because we want to preserve `.server.js` convention. It's not possible to define entries that contain dots using CLI args because `tsup` thinks that we want to create a nested object (it splits the given value by a dot and it's not possible to "escape" it anyhow)